scripts/cat/cats.py

```diff
@@ -480,6 +467,35 @@ def __eq__(self, other):
     def __hash__(self):
         return hash(self.ID)
 
+    @property
+    def dead(self) -> bool:
+        if self.status.group and self.status.group.is_afterlife():
+            return True
+        return False
+
+    @dead.setter
+    def dead(self, die: bool):
+        if die:
+            if self.status.group and self.status.group.is_afterlife():
+                return
+            self.status.send_to_afterlife()
+
+    @property
+    def dead_for(self) -> int:
+        count = 0
+        for entry in self.status.group_history:
+            if entry.get("group") in (
+                CatGroup.STAR_CLAN,
+                CatGroup.UNKNOWN_RESIDENCE,
+                CatGroup.DARK_FOREST,
+            ):
+                count += entry.get("moons_as")
+        return count
```

Comment:
```suggestion
        return sum(entry.get("moons_as") for entry in self.status.group_history if entry.get("group") in (CatGroup.STAR_CLAN, CatGroup.UNKNOWN_RESIDENCE, CatGroup.DARK_FOREST))
```
Double-check this one in-game, but I think that's right.